package.json

```diff
@@ -16,6 +16,7 @@
     "test:runtime:bundlers": "node test/runtime-integration/bundlers.cjs",
     "test:runtime:node": "node test/runtime-integration/node.cjs"
   },
+  "packageManager": "yarn@3.0.0",
```

Comment:
Q: Is this for `corepack`?

Reply:
Yes, `corepack` will launch `yarn@3.0.0` which will check `yarnPath` and use that one if it doesn't match